### PR TITLE
[KMSv2] enable presubmit for release-1.27

### PR DESCRIPTION
**Pull request by aramase**
fixes https://github.com/kubernetes/test-infra/issues/29044

/sig auth
/triage accepted
/assign @enj

### CODE REVIEW

File: config/jobs/kubernetes/sig-auth/sig-auth-encryption-at-rest.yaml [hunk 1]
```diff
@@ -14,7 +14,8 @@ presubmits:
     run_if_changed: 'staging/src/k8s.io/apiserver/pkg/storage/value/|staging/src/k8s.io/kms/|staging/src/k8s.io/apiserver/pkg/server/options/encryptionconfig/|test/e2e/testing-manifests/auth/encrypt/'
     path_alias: k8s.io/kubernetes
     branches:
-    - ^master$ # TODO(aramase): enable for release branches
+    - ^master$
+    - release-1.27
```
Comment (enj) [inline config/jobs/kubernetes/sig-auth/sig-auth-encryption-at-rest.yaml:18]: Can we use a regex to make this match 1.27 and greater?

Comment (aramase) [reply config/jobs/kubernetes/sig-auth/sig-auth-encryption-at-rest.yaml]: updated it! the regex will handle until `release-1.99`. We can update it after that 😉

Comment (aramase) [reply config/jobs/kubernetes/sig-auth/sig-auth-encryption-at-rest.yaml]: added regex for until `release-1.99`. We'll need to update this for 2.0 😄

**Pull request merged**